engine/src/main/java/org/terasology/engine/rendering/opengl/OpenGLMesh.java

```diff
@@ -86,7 +86,7 @@ public void render() {
             }
             GL30.glBindVertexArray(0);
         } else {
-            logger.error("Attempted to render disposed mesh: {}", getUrn());
+            logger.atError().addArgument(() -> getUrn()).log("Attempted to render disposed mesh: {}");
```

Comment:
Only evaluated in unexpected cases. Evaluated method is simple getter for member access, so impact neglectable.
```suggestion
            logger.error("Attempted to render disposed mesh: {}", getUrn()); //NOPMD
```